### PR TITLE
Fluff items for Linda Clark

### PR DESCRIPTION
**Pull request by FlattestGuitar**
Adds a lovely beret and a lovely green hoodie.

Londa Clolk

Also adds a lovely green hood, which I don't think I have to mention, shouldn't be touched with the database.

Also has vox sprites.

:bird: :gun: 

  

  

 

I really hate taking all those screenshots, can someone do them for me the next time?

### CODE REVIEW

**Comment by Jaywings** [comment]
greeeeeeeeeeen

**Comment by EpicCharger** [comment]
10/10 Would be a Vox in a hoodie again.

**Comment by sasanek12** [comment]
grüüüüüüüüüüüüüüüüün

**Comment by FlattestGuitar** [comment]
Where do I press to speedmerge?

**Comment by LittleBigKid2000** [comment]
She's got a lovely butt to go with that lovely beret and lovely hoodie.

**Comment by KasparoVy** [comment]
Thanks a bunch for keeping up the Vox sprite parity, much appreciated!

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -326,6 +326,11 @@
 		if(H.head == src)
 			H.slurring = max(3, H.slurring) //always slur
 
+/obj/item/clothing/head/beret/fluff/linda //Epic_Charger: Linda Clark
+	name = "Green beret"
+	desc = "A beret, an artists favorite headwear. This one has two holes cut on the edges."
```
Comment (TheDZD) [inline code/modules/customitems/item_defines.dm]: "an artists" is grammatically incorrect. If you mean a single artist, then it should be "an artist's", if you mean artists as a group, it should be "the favorite headwear of artists".

**Comment by Fox-McCloud** [comment]
@Necaladun 

Need confirmation.

File: code/modules/customitems/item_defines.dm (hunk 1 above)
Comment (LittleBigKid2000) [inline code/modules/customitems/item_defines.dm]: Me fail english! Thats unpossible?

Comment (FlattestGuitar) [inline code/modules/customitems/item_defines.dm]: I didn't pay attention to the stuff Epic sent me and I just copypasted. It was late, don't judge.

**Comment by KingPhilipIII** [comment]
Totally going to hunt you down with a stunprod and cuffs, drag you into maintenance, then just steal your hoodie and run.

**Comment by Jaywings** [comment]
Thank god nobody can steal my Jay's tail.

**Comment by KingPhilipIII** [comment]
I can steal your tail if I pay someone to help me surgically steal your torso.

**Comment by Dumbdumn5** [comment]
I'd steal @Jaywings tail if it didn't mean he would steal my rusted chassis at the same time. Mindswap has some drawbacks.

**Comment by KingPhilipIII** [comment]
I was thinking along the lines of sawing off his head, paying a surgeon to saw off mine and reattach it to his body.

**Comment by KingPhilipIII** [comment]
I know it's possible because once as the CMO I ended the round as a human with a tail. One of my vulpkanin surgeons was bored.

**Comment by Fox-McCloud** [comment]
Independently confirmed with Mel.

**Pull request merged**